packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -0,0 +1,297 @@
+import {helpers, ShopperLogin, ShopperLoginTypes} from 'commerce-sdk-isomorphic'
+import jwtDecode from 'jwt-decode'
+import {ApiClientConfigParams} from '../hooks/types'
+import {BaseStorage, LocalStorage, CookieStorage} from './storage'
+
+type Helpers = typeof helpers
+interface AuthConfig {
+    shortCode: string
+    organizationId: string
+    clientId: string
+    siteId: string
+    redirectURI: string
+    proxy: string
+}
+
+interface JWTHeaders {
+    exp: number
+    iat: number
+}
+
+// this type is slightly different from ShopperLoginTypes.TokenResponse, reasons:
+// 1. TokenResponse is too generic (with & {[key:string]: any}), we need a more
+//    restrictive type to make sure type safe
```

Comment:
Do we know why that type had `{[key:string]: any}`? I believe that is typically done when there is the possibility of customization for that response.

Reply:
I'm not 100% sure. but my guess is that that's not the expected output, it could be that isomorphic code generation script append `{[key:string]: any}` to all API response types to allow `c_` properties. But for the SLAS endpoints, i don't think it makes sense.

Reply:
Yeah.. that is probably the case. So just be clear, hooks aren't be implemented for slas api's?